src/controllers/livetv/livetvchannels.js

```diff
@@ -117,6 +118,9 @@ export default function (view, params, tabContent) {
         loading.show();
         isLoading = true;
         const query = getQuery();
+        const hasFilters = getFilterStatus(query);
+        setFilterStatus(context, hasFilters);
```

Comment:
Maybe we should change the function to just accept the query directly instead of calling `getFilterStatus` immediately followed by `setFilterStatus` every time.